src/multibody/parser/urdf-with-geometry.hpp

```diff
@@ -1,98 +0,0 @@
-//
```

Comment:
I dont know why git has not been able to track the modification on this file. Maybe you forget to explicitly use git move to move the file to src/parsers/urdf/geometry.hxx ? I think it is good to keep the history of this file. Could you say to git to move it?